app/ingestion/salary_mapping.json

```diff
@@ -0,0 +1,28 @@
+{
+  "mappings": {
+    "salary": {
```

Comment:
similar suggestions about mappings - it is better to make the mappings be more general (ability to support all the use cases in future) than more restrictive. If we are restrictive, we would have to reindex the documents in future to support use cases which require for example free-form search for fields which are marked only as keyword.